src/Logging/src/DynamicSerilogBase/SerilogHostBuilderExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Hosting;
+using Serilog;
+using System;
+
+namespace Steeltoe.Extensions.Logging.DynamicSerilog
+{
+    public static class SerilogHostBuilderExtensions
+    {
+        /// <summary>
+        /// Configure Serilog as the <see cref="IDynamicLoggerProvider"/> to enable dynamically controlling log levels via management endpoints
+        /// </summary>
+        /// <param name="hostBuilder">The <see cref="IHostBuilder"/> to configure</param>
+        /// <param name="configureLogger">The delegate for configuring the <see cref="DynamicLoggerConfiguration" /> that will be used to construct a <see cref="Serilog.Core.Logger" /></param>
+        /// <param name="preserveStaticLogger">Indicates whether to preserve the value of <see cref="Log.Logger"/>.</param>
+        /// <returns>The <see cref="IHostBuilder"/></returns>
+        public static IHostBuilder AddDynamicSerilog(
```

Comment:
Perhaps intended ... one method name (below) has `Console` and one doesn't (this)

Reply:
I could be wrong, but I thought this integration wasn't actually exclusively for console logging, so that's why I dropped it... maybe @hananiel can weigh in here to clarify ~ I would agree that it should be there if that is the case, and it should probably be made consistent across the board in this PR

Reply:
@TimHess that is right - it isn't for exclusively for console logging. It allows you to have a variety of sinks.